.travis.yml

```diff
@@ -26,6 +26,7 @@ before_script:
   - export CXX=g++-5 CC=gcc-5  # override travis defaults
 
 script:
+  - bash scripts/install/install_geographiclib.bash
```

Comment:
@leokoppel Can you take a look at the `.travis.yml`? I took the bash script for installing GeographicLib and put it here.